archetypes.js

```diff
@@ -0,0 +1,1484 @@
+const emptyBuild = {
+  position: 'None',
+  shortPosition: 'None',
+  archetype: 'None',
+  baseTpe: 0,
+  strengthMin: 0,
+  strengthMax: 0,
+  agilityMin: 0,
+  agilityMax: 0,
+  armMin: 0,
+  armMax: 0,
+  intelligenceMin: 0,
+  intelligenceMax: 0,
+  throwingAccuracyMin: 0,
+  throwingAccuracyMax: 0,
+  tacklingMin: 0,
+  tacklingMax: 0,
+  speedMin: 0,
+  speedMax: 0,
+  handsMin: 0,
+  handsMax: 0,
+  passBlockingMin: 0,
+  passBlockingMax: 0,
+  runBlockingMin: 0,
+  runBlockingMax: 0,
+  enduranceMin: 0,
+  enduranceMax: 0,
+  kickPowerMin: 0,
+  kickPowerMax: 0,
+  kickAccuracyMin: 0,
+  kickaccuracyMax: 0
+};
+
+const safetyCenterFielder = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'Center Fielder',
+  baseTpe: 466,
+  strengthMin: 40,
+  strengthMax: 70,
+  agilityMin: 70,
+  agilityMax: 95,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 40,
+  intelligenceMax: 80,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 50,
+  tacklingMax: 75,
+  speedMin: 70,
+  speedMax: 100,
+  handsMin: 55,
+  handsMax: 75,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 70,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const safetyPlayermaker = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'Playmaker',
+  baseTpe: 456,
+  strengthMin: 40,
+  strengthMax: 70,
+  agilityMin: 70,
+  agilityMax: 95,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 50,
+  intelligenceMax: 80,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 50,
+  tacklingMax: 75,
+  speedMin: 65,
+  speedMax: 95,
+  handsMin: 55,
+  handsMax: 85,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 65,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const safetyRunSupport = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'Run Support',
+  baseTpe: 426,
```

Comment:
386 template, 50 available, shouldn't it be 436?

Reply:
(MAX: 80) Strength: 50 (50 TPE)
(MAX: 90) Agility: 60 (70 TPE)
(MAX: 25) Arm: 1 (1 TPE)
(MAX: 80) Intelligence: 45 (45 TPE)
(MAX: 25) Throwing Accuracy: 1 (1 TPE)
(MAX: 85) Tackling: 60 (70 TPE)
(MAX: 90) Speed: 55 (60 TPE)
(MAX: 70) Hands: 45 (45 TPE)
(MAX: 25) Pass Blocking: 1 (1 TPE)
(MAX: 25) Run Blocking: 1 (1 TPE)
(MAX: 100) Endurance: 65 (80 TPE)
(MAX: 25) Kick Power: 1 (1 TPE)
(MAX: 25) Kick Accuracy: 1  (1 TPE)

50+70+1+45+1+70+60+45+1+1+80+1+1 = 426

Reply:
somehow forgot the tpe translation my b! resolving every point above about the same topic